ci/post/main.py

```diff
@@ -22,6 +22,7 @@
 # LINUX_RESULT: bool, if the linux builds were sucessfully uploaded
 # WINDOWS_RESULT: bool, if the windows builds were successfully uploaded
 
+from curses import window
```

Comment:
I guess this is another stray auto-import from VSCode?
You can disable that. I think the option is called [`autoImportCompletions`](https://code.visualstudio.com/docs/python/settings-reference#_python-language-server-settings).